package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@nimblehq/infra-template",
+  "version": "0.1.1",
```

Comment:
Should we change this version to `1.0.0` as well?

Reply:
Added the PR https://github.com/nimblehq/infrastructure-templates/pull/44